extensions/resteasy-reactive/quarkus-resteasy-reactive-jsonb/deployment/src/test/java/io/quarkus/resteasy/reactive/jsonb/deployment/test/SimpleJsonResource.java

```diff
@@ -21,7 +21,6 @@
 
     @GET
     @Path("/person")
-    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
The PR doesn't introduce any new tests, it just reuses the existing ones and removes `@Produces(MediaType.APPLICATION_JSON)` to ensure everything works properly